docs/en/cloud/AWS-EMR.md

```diff
@@ -31,93 +31,192 @@ Blob Store.
+or click on the `Hardware` tab to see the master and worker details.
+
+![emr_console_waiting]({{ '/img/emr_console_waiting.png' | relativize_url }})
+
+Clicking on the master instance group will show you the public DNS.
+
+![emr_console_public_dns]({{ '/img/emr_console_public_dns.png' | relativize_url }})
+
+SSH into the master instance using the key pair provided in the previous command.
+Use `hadoop` as the username.
 ```console
-$ alluxio runTests
+ssh -i /path/to/keypair.pem hadoop@masterPublicDns
 ```
 
-Alluxio is installed in `/opt/alluxio/` by default.
-Hive and Presto are already configured to connect to Alluxio.
-The cluster also uses AWS Glue as the default metastore for both Presto and Hive.
-This will allow you to maintain table definitions between multiple runs of the Alluxio cluster.
+If a security group isn't specified in the `create-cluster` command,
+the default EMR security group created for you will **not** allow inbound SSH.
+In order to continue, you will need to edit the security group and open port 22.
+See more details [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/authorizing-access-to-an-instance.html#add-rule-authorize-access).
 
-See the below sample command for reference.
+  {% endcollapsible %}
+{% endaccordion %}
 
+{% accordion setup test %}
+  {% collapsible Test that Alluxio is running %}
+Once inside the master instance, run the following command to run a series of basic tests to ensure Alluxio can read and write files.
 ```console
-$ aws emr create-cluster \
---release-label emr-5.25.0 \
---instance-count 3 \
---instance-type m4.xlarge \
---applications Name=Presto Name=Hive \
---name 'Test Cluster' \
---bootstrap-actions \
-Path=s3://alluxio-public/emr/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-emr.sh,\
-Args=[s3://test-bucket/path/to/mount/] \
---configurations https://alluxio-public.s3.amazonaws.com/emr/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-emr.json \
---ec2-attributes KeyName=admin-key
+sudo runuser -l alluxio -c "/opt/alluxio/bin/alluxio runTests"
 ```
 
-> Note: The default Alluxio Worker memory is set to 1/3 of the physical memory on the instance.
-If a specific value is desired, set `alluxio.worker.memory.size` in the provided
-`alluxio-site.properties` or in the additional options argument.
+  {% endcollapsible %}
+{% endaccordion %}
+
+{% accordion setup details %}
+  {% collapsible Cluster details %}
+Using this boostrap script, Alluxio is installed in `/opt/alluxio/` by default.
+Hive and Presto are already configured to connect to Alluxio.
+The cluster also uses AWS Glue as the default metastore for both Presto and Hive.
+This will allow you to maintain table definitions between multiple runs of the Alluxio cluster.
+
+The default Alluxio worker is allocated a third of the instance's available memory.
+If the instance type has less than 20GB of memory, you will need to make your own copy of the `alluxio-emr.sh` script
+and update the divisor value at `mem_div=3`.
```

Comment:
why are we telling people to do this? It will still work fine even with less than 20GB...

It should be clear this is a recommendation and not a requirement.

Reply:
this was described (poorly) in the tutorial. so can i omit this entire sentence?

Reply:
I agree.